.taskcluster.yml

```diff
@@ -53,6 +53,28 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("test_enums_workspace")}
```

Comment:
Let's add it to the `lint_test_task` task instead, so we don't pay the setup price twice.